.github/workflows/mend-pr-scan.yml

```
@@ -0,0 +1,37 @@
+name: Mend PR Security Scan
+
+on:
+  pull_request:
+    types: [opened, synchronize, reopened]
+
+jobs:
+  mend-scan:
```

Because this is an open source repo, this gets forked a lot by the community. This means that anyone who newly forks this repo or synchronizes their existing fork with our latest changes will also get this workflow, even though it won't work for them because they won't have the corresponding secrets enabled in their org.

While this isn't super elegant, I believe the canonical way to fix this is to add something like:

```suggestion
  mend-scan:
    if: github.repository == ‘hashicorp/terraform’
```

---

This PR is part of our automated deployment process that applies a generic Mend workflow template across multiple repositories (both HashiCorp internal and community repos).

Since this is an automated PR from our deployment bot, I'd suggest:

* Merge this PR to enable the baseline Mend scanning functionality
* Follow-up enhancement: Add the repository condition check in a subsequent PR to prevent issues for   community forks

This approach keeps our deployment automation clean while addressing the fork scenario